res/external-mqtt-trigger/configuration.yaml

```diff
@@ -49,5 +59,5 @@ Trigger:
     QoS: 0 # Quality of Service 0 (At most once), 1 (At least once) or 2 (Exactly once)
     Retain: true
     SkipCertVerify: false
-    SecretName: "mqtt-trigger" 
+    SecretName: "mqtt" 
     AuthMode: "none"
```

Comment:
Issues say the `AuthMode` should be `usernamepassword`

Reply:
@lenny-intel Since the username/password set empty on this file, to avoid the startup error I plan to override the variables on the compose file.

Reply:
The compose file will be updated [here](https://github.com/edgexfoundry/edgex-taf/blob/28b46344f0e044d7fd6d5e281fe0f2244ee83f6b/TAF/utils/scripts/docker/get-compose-file.sh#L97-L106)

Reply:
OK